### DISCUSSION TITLE
Dimension of the solar radiation results

### DISCUSSION THREAD

**Discussion by hxTrans** [UMEP-dev/UMEP#497] [General]
Hello, may I know the dimension of the solar radiation results. If my input is a DSM 1 arc-second, the solar radiation will be displayed in grids of 30m*30m. Is that correct？

**Comment by biglimp** [comment]
Depends on where you are on the globe. 1 arc-second is getting smaller as you travel north from the equator (30m at the equator). Change your CRS to a meter-based system e.g. UTM to see your pixel resolution.

**Reply by hxTrans** [reply]
Noted! Thank you for the reply. 
One more question: Can SEBE calculate solar irradiance on vertical building facade? I can see the results in SEBE (visualisation) , but don't know how to extract the value of solar irradiance on the facade area.

**Reply by biglimp** [reply]
Yes, it can. Check publication and tutorial found on our website.

**Reply by hxTrans** [reply]
Great! Could you provide some instructions or guidance on how to export the value of solar irradiance on the vertical facade in QGIS? 
For example, can I subdivided the facade into patches and obtain the irradiance of the centroid of each patch? 

![Capture](https://user-images.githubusercontent.com/54798910/217127311-974b743f-62b5-412d-8b73-9f6d9647091d.JPG)

**Reply by biglimp** [reply]
The format of the current output from the model is found here: https://umep-docs.readthedocs.io/en/latest/processor/Solar%20Radiation%20Solar%20Energy%20on%20Building%20Envelopes%20(SEBE).html#solar-radiation-solar-energy-on-building-envelopes-sebe

We are currently working with an update of the format. If you have any preferences of suitable format, please let us know and we will take it into consideration.

**Reply by hxTrans** [reply]
Hi biglimp, 

Sorry for the late follow-up. I found the txt file containing the irradiance of building facades.
![image](https://user-images.githubusercontent.com/54798910/223613621-9a8da77f-5404-4646-bff7-8ab4a3aabfc5.png)
![image](https://user-images.githubusercontent.com/54798910/223613661-a1c13c9f-e6a9-4b47-b055-f0168846f013.png)

Is there any way that I can obtain the geolocation of each pixel on building facades according to the 'row' and 'column' ? I have read the paper of Lindberg, F. et all. The simulation is very accurate. Therefore, I would like to have the numeric results besides visualization. 

Many thanks in advance!

**Reply by biglimp** [reply]
@nilswallenberg , didnt you just write some conversion between rows and columns to geographical coordinates?

**Reply by biglimp** [reply]
To recalculate to geographic coordinates you need to know the dimensions and pixel size from the original raster, e.g. your DSM. The calculations looks at below: 

GT(0) x-coordinate of the upper-left corner of the upper-left pixel.
GT(1) w-e pixel resolution / pixel width.
GT(2) row rotation (typically zero).
GT(3) y-coordinate of the upper-left corner of the upper-left pixel.
GT(4) column rotation (typically zero).
GT(5) n-s pixel resolution / pixel height (negative value for a north-up image).

minx = gt[0]
maxy = gt[3]             
ywall = abs(rowwall - (maxy - gt[5]/2))
xwall = (colwall + (minx  - gt[1]/2))
